docs/shortcuts.md

```diff
@@ -106,10 +106,11 @@ Get the most out of Sequel Pro by remembering the following keyboard shortcuts.
 | `⇧ + ↩`, `⇧ ⇥`, `⇧ Double-Click` | Insertion full schema reference              |
 
 Keep Window Open after Insertion
+
```

Comment:
Did you mean to add in a newline here?

Reply:
Yes, it's now appended, in the origin they were also among each other.

Reply:
Hmmmm I think I like the before a little better for that one line. Perhaps we could add the word "by" between inserting and holding down to make it a clearer sentence?

Reply:
Left aligning overall looks good though!

Reply:
That is a great alternative! Rebased. :-)